packages/blade/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -34,7 +40,7 @@ type ProgressBarCommonProps = {
    * Sets the size of the progress bar.
    * @default 'small'
```

Comment:
We should add a small note about the runtime check we have added. 

Like which variant cannot be used with which size.